CRM/Contact/Form/Contact.php

```diff
@@ -869,8 +869,9 @@ public function postProcess() {
       $params['preferred_communication_method'] = 'null';
     }
 
-    $group = $params['group'] ?? NULL;
-    if (!empty($group) && is_array($group)) {
+    $group = CRM_Utils_Array::value('group', $params);
+    if (!empty($group)) {
```

Comment:
CRM_Utils_Array::value is being mercilessly removed everywhere. So it should be the original line `$group = $params['group'] ?? NULL;`